github_connector/models/github.py

```diff
@@ -74,12 +74,14 @@
 
 class Github(object):
 
-    def __init__(self, github_type, login, password, max_try):
+    def __init__(self, github_type, login, password, max_try, token):
```

Comment:
Don't change signature this way, but make it optional for being compatible with prior extensions:
```suggestion
    def __init__(self, github_type, login, password, max_try, token=None):
```